src/config/default.properties

```diff
@@ -15,6 +15,7 @@ testdb_container=montagu-api-test-db
 
 docker_host=tcp://localhost:2375
 
+app_auth=true
```

Comment:
Is it worth also setting app_auth=true in the docker.properties file? The latter being production values, these being default or local ones. An extra guard against app_auth being set to true in a deploy!